recaptcha_enterprise/demosite/README.md

```diff
@@ -0,0 +1,6 @@
+# Demosite - Google Cloud reCAPTCHA Enterprise
+
+Google [Cloud reCAPTCHA Enterprise](https://cloud.google.com/recaptcha-enterprise) helps protect your website from fraudulent activity, spam, and abuse without creating friction.
+
+This application demonstrates how to use integrate your client and server code with reCAPTCHA Enterprise - Java Client library.
```

Comment:
nit: Wording gets a bit clumsy here: "demonstrates how to use integrate your client"

Just delete "use"

Reply:
Thanks for catching. Done.